docs/install-apps.md

```diff
@@ -11,6 +11,7 @@ The following are the parameters supported by this goal in addition to the [comm
 | appsDirectory | The directory where the application files should be copied. The default value is `dropins`.  | No |
 | stripVersion | Strip artifact version when copying the application to Liberty runtime's application directory. The default value is `false`. | No |
 | installAppPackages | The Maven packages to copy to Liberty runtime's application directory. One of `dependencies`, `project` or `all`. The default is `dependencies`. | No |
+| looseConfig | Generate a loose configuration file representing the Maven project package and copy to Liberty server's `apps` or `dropins` directory. The default value is `false`.  This parameter will be ignored if installAppPacakges is set to `dependencies`, or the project packaging type is neither `war` nor `liberty-assembly`.  | No |

```

Comment:
I will update the doc to use "loose application" and the parameter change from "looseConfig" to "looseApplication"